src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const propsDividedToStrings = sourceString.split(';');
+
+  const resultObject = {};
+
+  propsDividedToStrings.forEach((propertyPair) => {
+    if (propertyPair.includes(': ')) {
+      const propAndValue = propertyPair.split(': ')
+        .map((proper) =>
+          proper.trim());
+
+      resultObject[propAndValue[0]] = propAndValue[1];
+    }
+  });
```

Comment:
You can create an object with `reduce` method instead of creating an empty object and adding properties in `forEach`.